lib/kubeclient/common.rb

```diff
@@ -204,6 +204,16 @@ def retrieve_all_entities(entity_types)
         end
       end
 
+      def proxy_url(type, service, port_name, namespace = '')
```

Comment:
@jimmidyson @oschreib yes the names should probably be (to be consistent with kubernetes):

    proxy_url(kind, name, port, namespace = '')